src/transaction/sign.ts

```diff
@@ -241,4 +246,18 @@ function sign(
   }
 }
 
-export default sign
+function signOffline(
+  wallet: Wallet,
+  txJSON: string,
+  options?: SignOptions,
+): SignedTransaction {
+  const {publicKey, privateKey} = wallet
+  return signWithKeypair(
+    null,
+    txJSON,
+    {publicKey, privateKey},
+    options,
+  )
+}
```

Comment:
This should really be implemented in the `Wallet` instead of here, but looks like the typing makes that hard. Maybe for now just add a `TODO` placeholder

Reply:
Yes, I agree. The reason why I put it here is `signOffline()` call flow is exactly the same as `sign()` except it doesn't call `checkFee()`. They both call `signWithKeypair()` so I thought it would be best to keep `signOffline()` here too. I'll add a `TODO` placeholder here.